specs/agents/tracing-api-otel.md

```diff
@@ -5,6 +5,8 @@
 Agents MAY provide a bridge implementation of OpenTelemetry Tracing API following this specification.
 When available, implementation MUST be configurable and should be disabled by default when marked as `experimental`.
 
+Agents MAY add an explicit agent configuration to enable/disable the bridge. If an agent implements such a configuration, the configuration name MUST be `ELASTIC_APM_OPENTELEMETRY_BRIDGE_ENABLED`.
```

Comment:
Should we use the canonical configuration item notation `opentelemetry_bridge_enabled` here?
Or if we only want to support environment variables for this, let's state it explicitly.

Reply:
Very good point, thanks!

I think we should go with the canonical notation `opentelemetry_bridge_enabled` - this is not environment variable-only.

So updated to `opentelemetry_bridge_enabled`.